Reneiski_Homework1/Reneiski_Homework1/Program.cs

```diff
@@ -1,28 +1,279 @@
-using Reneiski_Homework1.Lecture5;
-using WorkerNamespace;
+﻿using Reneiski_Homework1.Lecture9;
 
 namespace Reneiski_Homework1
 {
     public class Program
     {
         static void Main(string[] args)
         {
-            Zavod Belaz = new Zavod(new Worker(), "Belaz", "The best of the best!");
-            Worker sasha = new Worker("Alex", "Ren", 41, "QA");
-            Worker ivan = new Worker("Ivan", "Ivanov", 54, "Director");
-            Worker semen = new Worker("Semen", "Semenov", 51, "QA");
-
-            Belaz.SetWorker(sasha);
-            Belaz.SetWorker(ivan);
-            Belaz.SetWorker(semen);
-
-            Belaz.ShowAllEmployeeInfo();
-            Belaz.ShowEmployeeNameInfo();
-            Belaz.ShowEmployeeAgeInfo();
-            Belaz.ShowEmployeePositionInfo();
-            Belaz.ShowEmployeeNumber();
-            Belaz.ShowEmployeeByPosition();
-            Belaz.ShowEmployeeWork();
+            //Lists for Tasks
```

Comment:
could you please move all methods that contains tasks logic into separate class with static methods? and in main just call these static methods?

Reply:
Fixed